core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -61,6 +61,8 @@ var ExplorationEditorFeedbackTab = function() {
       suggestionDescription));
     expect(await matchingRow.isDisplayed()).toBe(true);
     await matchingRow.click();
+    await waitFor.elementToBeClickable(
```

Comment:
Instead of this, can you use `actions.click(..)` in line 67?

Reply:
Done